src/IW/Effects/Cabal.hs

```diff
@@ -32,34 +33,40 @@ instance MonadCabal App where
 
 type WithCabal env m = (MonadDownload m, WithLog env m, WithError m)
 
-{- | This function may throw anyone of the errors inherited by the use of @downloadFile@
-defined in @IW.Effects.Download@. We are using @parseGenericPackageDescriptionMaybe@
-which will return @Nothing@ on an unsuccessful parse.
+{- | This function takes a 'Repo' and either returns @['Category']@, or
+throws a 'CabalParseError' error. This function may also throw any one of the
+errors inherited by the use of 'downloadFile' defined in @IW.Effects.Download@.
```

Comment:
In haddock, you can actually link to the module: https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810751008
```suggestion
errors inherited by the use of 'downloadFile' defined in "IW.Effects.Download".
```

Reply:
@vrom911 Nice!